### ISSUE TITLE
Internal error: `TypeError: type.__new__() argument 3 must be dict, not ShellMutableMap`

### ISSUE THREAD

**Issue by Zac-HD** [pschanely/CrossHair#304]
```python-traceback
  File ".../hypothesis/extra/pytz.py", line 43, in timezones
    all_timezones = [pytz.timezone(tz) for tz in pytz.all_timezones]
  File ".../hypothesis/extra/pytz.py", line 43, in <listcomp>
    all_timezones = [pytz.timezone(tz) for tz in pytz.all_timezones]
  File ".../pytz/__init__.py", line 184, in timezone
    _tzinfo_cache[zone] = build_tzinfo(zone, fp)
  File ".../pytz/tzfile.py", line 118, in build_tzinfo
    cls = type(zone, (DstTzInfo,), dict(
  File ".../crosshair/libimpl/builtinslib.py", line 4657, in _type
    return type(*map(deep_realize, a))
TypeError: type.__new__() argument 3 must be dict, not ShellMutableMap
on backend='crosshair' but overran under backend='hypothesis'
```

via [this CI run](https://github.com/HypothesisWorks/hypothesis/actions/runs/10543116903/job/29210706458?pr=4034#step:6:737) for https://github.com/HypothesisWorks/hypothesis/pull/4034 (`test_datetimes_stay_within_naive_bounds`)

**Comment by pschanely** [comment]
Oh great, thanks! Should be able to fix this week.

**Comment by pschanely** [comment]
A little late, but have a fix in 0.0.71!
